views/DashboardContainer.tsx

```diff
@@ -39,8 +46,29 @@ class DashboardContainer extends React.PureComponent<DashboardContainerProps> {
       windSensors.data.map((data) => data.windSensors[1].speed),
     ];
 
+    const totalTripDistance = this._computeTotalTripDistance(gpsDistanceData[0], gpsDistanceData[1])
+
     return (
       <div>
+        <Grid
+          container spacing={0}
+          direction={"row"}
+          justifyContent={"center"}
+          alignItems={"center"}
+          >
+          <Grid container justifyContent="center" item xs={3}>
+            <SingleValueChart title="Distance" data={totalTripDistance} unit="km" />
+          </Grid>
+          <Grid container justifyContent="center" item xs={3}>
+            <SingleValueChart title="Heading" data={this.props.gps.data.at(-1)?.heading} unit="°"/>
+          </Grid>
+          <Grid container justifyContent="center" item xs={3}>
+            <SingleValueChart title="Latitude" data={this.props.gps.data.at(-1)?.latitude.toFixed(2)} unit="°"/>
+          </Grid>
+          <Grid container justifyContent="center" item xs={3}>
+            <SingleValueChart title="Longitude" data={this.props.gps.data.at(-1)?.longitude.toFixed(2)} unit="°"/>
+          </Grid>
+        </Grid>
```

Comment:
I know it's not ideal, but for now, I think it's best to remove all the instances `SingleValueChart` (we'll come back to this in the future). We already show the information we need on the map itself, e.g. distance travelled and heading, and as we discussed, `latitude` and `longitude` isn't as useful to show.

I'll leave this open to discussion however, if you have strong feelings about keeping these charts.

Reply:
done
commented out